apiserver/Cargo.toml

```diff
@@ -5,12 +5,17 @@ edition = "2018"
 publish = false
 license = "Apache-2.0 OR MIT"
 
+[features]
```

Comment:
Sure! So, in this case the `agent` needs to communicate with the `apiserver` using the code in `apiserver::client`.

So in the `agent`'s `Cargo.toml`, we add a dependency on `apiserver` but only enable the `client` feature. In this case, when the compiler builds the `agent` crate, it will not compile in the `apiserver`'s server code, which means we can't accidentally use code which is meant only for the server component. It also improves compile times.